lib/turbo/system_test_helper.rb

```diff
@@ -0,0 +1,20 @@
+module Turbo::SystemTestHelper
+  extend ActiveSupport::Concern
+
+  # Delay until every `<turbo-cable-stream-source>` element present in the page
+  # is ready to receive broadcasts
+  #
+  #   test "Page receives new Messages broadcast over Action Cable" do
+  #     visit messages_path
+  #     wait_for_turbo_cable_stream_sources
+  #
+  #     # create a new Message, then broadcast...
+  #
+  #     assert_text "Hello from Action Cable"
+  #   end
+  def wait_for_turbo_cable_stream_sources(**options, &block)
```

Comment:
I don't love this name, as the name itself is composed mostly of implementation details. To be honest, it's annoying and tedious to have to wait at all.

What would be _really_ nice (ergonomically) would be built-in support for Capybara's actions like `visit`, `click_link`, `click_button`, etc. We _could_ override those methods like:

```ruby
def visit(...)
  super
  wait_for_turbo_cable_stream_sources
end
```

That feels pretty invasive and presumptive, but it papers over the tedium.

Regardless, documenting the call is something we'll need to do regardless. The most important part of documentation is the name itself, so we'll want to come up with something that's an improvement on `wait_for_turbo_cable_stream_sources`.

Reply:
I've pushed up a change to extend `#visit`. Does that feel like a good balance between surprise, risk, and convenience?